extensions/chathistory.md

```diff
@@ -0,0 +1,88 @@
+---
+title: IRCv3 chathistory extension
+layout: spec
+work-in-progress: true
+copyrights:
+  -
+    name: "Evan Magaliff"
+    period: "2017"
+    email: "evan@muffinmedic.net"
+---
+## Notes for implementing work-in-progress version
+This is a work-in-progress specification.
+
+Software implementing this work-in-progress specification MUST NOT use the unprefixed `CHATHISTORY` command. Instead, implementations SHOULD use the `draft/CHATHISTORY` command to be interoperable with other software implementing a compatible work-in-progress version.
+
+The final version of the specification will use an unprefixed command.
+
+## Description
+This document describes the format of the `chathistory` extension. The `chathistory` extension uses the [chathistory][batch/chathistory] batch type, adding a client-side command for requesting `chathistory` content from the server.
+
+The `chathistory` extension adds an optional `draft/msgid` to the `chathistory` batch reply.
+
+Clients MUST support the [`batch`][batch] and [`server-time`][server-time] capabilities. Clients SHOULD support the [`draft/labeled-response`][draft/labeled-response] and [`draft/msgid`][draft/msgid] capabilities.
```

Comment:
My understanding is that CAPS are negotiated before the ISUPPORT is sent - if the CAPs are not supported (as required by `MUST`), then `CHATHISTORY` should not appear in the 005. This is already standard behavior to my best of my knowledge.

Will update capability URL.